tools/reviewer/web/src/app/dashboard/reviews-panel/review/person-list/person-list.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Diff,
+  FirebaseService,
+  NotificationService,
+  ProtoService,
+  Status
+} from '@/shared';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-person-list',
+  templateUrl: './person-list.component.html',
+  styleUrls: ['./person-list.component.scss']
+})
+export class PersonListComponent {
+  @Input() diff: Diff;
+  @Input() property: string;
+  @Input() addToAttention: boolean = true;
```

Comment:
Perhaps it should be name `enableAddToAttention` or `showAddToAttention`?

Reply:
`enableAddToAttention` is good.